### PR TITLE
Next Video Loader Component

### PR DESCRIPTION
**Pull request by Lukass008**

### CODE REVIEW

File: src/components/Spinner/Spinner.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      top: size + 5
+    }
+
+    const playStyle = {
+      // don't ask why that numbers, just from testing component, all proportions tested with different sizes works fine
+      borderWidth: `${radius / 3}px 0 ${radius / 3}px ${radius / 3 * 2}px`,
+      top: radius / 3 * 2,
+      left: radius / 3 * 2,
+      marginLeft: radius / 10
+    }
```
Comment (scwe) [inline src/components/Spinner/Spinner.js]: Maybe all these styles can be moved to the constructor, we shouldn't need to recreate them on every render

File: src/components/Recommendations/Element/Element.js [hunk 2]
```diff
@@ -132,10 +132,15 @@ export default class RecommendationElement extends Component {
         src={replayIcon} />
     }
     if (this.props.state === RECOMMENDATION_STATE.PENDING) {
-      return <img
-        onClick={this.props.onPlay}
-        className='cWebSdK-Recommendations-Element__icon'
-        src={playIcon} />
+      const loaderWrapperStyle = {
```
Comment (wojciechAdaszynski) [inline src/components/Recommendations/Element/Element.js]: Move it outside the function

File: src/components/Spinner/Spinner.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+// COMPONENTS
+import { AUTO_PLAY_DURATION } from '../../constants'
+// STYLES
+import './Spinner.css'
+
+class Spinner extends Component {
+  render () {
+    const { radius, border, color, duration, ringColor, description, displayPlayIcon } = this.props
+    const size = 2 * radius
+    const innerSize = size - 2 * border
```
Comment (wojciechAdaszynski) [inline src/components/Spinner/Spinner.js]: all of it can be done in the contructor

**Comment by psmarcin** [comment]
@wojciechAdaszynski

**Pull request merged**